packages/gatsby-theme-vtex/src/sdk/buyButton/useBuyButton.ts

```diff
@@ -33,7 +34,7 @@ export const useBuyButton = (sku: Maybe<SKU>) => {
     // Item to be updated into the orderForm
     const orderFormItem = {
       id: Number(sku!.itemId),
-      quantity: 1,
+      quantity: getCurrentQuantity(),
```

Comment:
here it would be better if we received the quantity value via the ʻuseBuyButton` arg